hs_core/templates/pages/genericresource.html

```diff
@@ -1005,7 +1005,7 @@ <h3>Resource Specific</h3>
           {% for author in creators %}
               {
                   "id": "{{ author.id }}",
-                  "name": "{{ author.name|default:"" }}",
+                  "name": "{{ author.name|default:""|escapejs }}",
```

Comment:
Do we risk the same problem with organization?  Any reason not to use escapejs with organization?

Reply:
What about best_name in the last_changed_by block below?

Reply:
Yeah need to fix those too.

Reply:
@sblack-usu Made the suggested changes.